static/js/apps/biomed_nl/app.tsx

```diff
@@ -123,40 +241,102 @@ export function App(): ReactElement {
       });
   }
 
+  const sampleQuestions = [
```

Comment:
let's make these global variables outside the App definition

Reply:
done